docs/docs/themes/authoring-themes.md

```diff
@@ -0,0 +1,129 @@
+---
+title: How to author a theme
```

Comment:
```suggestion
title: How to Author a Theme
```
This should use Title Case to follow the [Gatsby Style Guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/).